### PR TITLE
Bump avalanchego dep to v1.9.7 and start v0.4.9 release cycle

### PR DESCRIPTION
**Pull request by aaronbuchwald**
This PR bumps the AvalancheGo dependency to v1.9.7 and starts the v0.4.9 release cycle.

### CODE REVIEW

**Comment by aaronbuchwald** [comment]
Need to fix compatibility.json: https://github.com/ava-labs/subnet-evm/actions/runs/3903711540/jobs/6668459190#step:6:539

File: cmd/simulator/go.mod [hunk 1]
```diff
@@ -15,7 +15,7 @@ replace github.com/ava-labs/subnet-evm => ../..
 
 require (
 	github.com/VictoriaMetrics/fastcache v1.10.0 // indirect
-	github.com/ava-labs/avalanchego v1.9.6 // indirect
+	github.com/ava-labs/avalanchego v1.9.7 // indirect
```
Comment (ceyonur) [inline cmd/simulator/go.mod:18]: We also need to update README (+ compatibility json).

Comment (ceyonur) [reply cmd/simulator/go.mod:18]: README seems out-of-sync for recent releases

Comment (aaronbuchwald) [reply cmd/simulator/go.mod:18]: Updated

**Pull request merged**